tracer/test/Datadog.Trace.TestHelpers/LogSettingsHelper.cs

```diff
@@ -16,6 +16,7 @@ namespace Datadog.Trace.TestHelpers
     internal class LogSettingsHelper
     {
         public static LogFormatter GetFormatter() => new(
+            new ImmutableTracerSettings(new TracerSettings()),
```

Comment:
nit: Probably don't want to bother with the telemetry stuff here (and various other places). Doesn't matter if you do, just reduce test overhead by a _tiny_ amount. 🤷 


```suggestion
            new ImmutableTracerSettings(new TracerSettings(null, Configuration.Telemetry.NullConfigurationTelemetry.Instance)),
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/4902/commits/69541684633cdfbab014eb59c5240334490343e0